tests/wallets2descriptors.rs

```diff
@@ -0,0 +1,120 @@
+#[cfg(feature = "wallet_file")]
```

Comment:
it looks every test here need the feature wallet_file, I think you can remove any instance of `#[cfg(feature = "wallet_file")]` and add a global `#![cfg(feature = "wallet_file")]`

note the `!` will apply to the current file while without it refer to the following element

Reply:
Ah finally. I tried for a while to find the right syntax for the whole file by searching the documentation, but couldn't make it work. 
The one you provided works at the first try.